test/routes/src/server/routes/t_login.js

```diff
@@ -0,0 +1,105 @@
+// load server/server.js via proxyquire to stub webpack-assets.json
+import proxyquire from 'proxyquire';
+
+// import login routes for testing
+import loginRoutes from '../../../../../src/server/routes/login';
+
+const routesStub = {
+  login: loginRoutes,
+  api: () => {},
+  github: () => {},
+  githubAuth: () => {},
+  universal: () => {},
+  // tell proxyquire not to try to load stubbed module
+  '@noCallThru': true
+};
+const stubDependencies = {
+  './routes/': routesStub
+};
+
+const app = proxyquire(
```

Comment:
Yeah, that makes sense and it's definitely not ideal to have to explicitly list all the routes we're not testing.  I've converted these tests to the isolated style.